apps/aens/src/aens.erl

```diff
@@ -0,0 +1,78 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2018, Aeternity Anstalt
+%%% @doc
+%%%    Resolve registered names
+%%% @end
+%%%-------------------------------------------------------------------
+
+-module(aens).
+
+%% API
+-export([resolve/2,
+         get_name_entry/1]).
+
+%%%===================================================================
+%%% Types
+%%%===================================================================
+
+-define(LABEL_SEPARATOR, <<".">>).
+
+%%%===================================================================
+%%% API
+%%%===================================================================
+
+-spec resolve(atom(), binary()) -> {ok, binary()} | {error, atom()}.
```

Comment:
Could you please specify further the type of the `Type` argument from `atom()` to `account_pubkey|oracle_pubkey`? It clarifies intended usage to the reader.